Resources/Prototypes/Entities/Markers/Spawners/Random/posters.yml

```
@@ -139,12 +139,12 @@
       - PosterLegit50thAnniversaryVintageReprint
       - PosterLegitFruitBowl
       - PosterLegitPDAAd
-      - PosterLegitEnlist
+    # - PosterLegitEnlist
```

YAMLlint (1.35.1)

[warning] comment not indented like content

(comments-indentation)